<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка файлов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="choose_svod_wg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сводная таблица</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_otchet_wg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Общий перечень отчётов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_prod_wg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Общий перечень продукции ОВНТ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_sintez_wg">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Список синтезов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать местоположение файлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_svod">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>601</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>label_svod</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_otchet">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_otchet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sintez">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_sintez</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_prod">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>label_prod</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Текущее местоположение фала</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_but">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
